src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,927 @@
+
+                    foreach (ViewItem item in colView.ContentContainer.Children.Where((item) => item is ViewItem))
+                    {
+                        if (item.BindingContext == null) continue;
+                        if (item.BindingContext == oldValue) item.IsSelected = false;
+                        else if (item.BindingContext == newValue) item.IsSelected = true;
+                    }
+
+                    SelectionPropertyChanged(colView, args);
+                },
+                defaultValueCreator: (bindable)=>
+                {
+                    var colView = (CollectionView)bindable;
+                    return colView.selectedItem;
+                });
+
+        /// <summary>
+        /// Binding Property of selected items list in multiple selection.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty SelectedItemsProperty =
+            BindableProperty.Create(nameof(SelectedItems), typeof(IList<object>), typeof(CollectionView), null,
+                propertyChanged: (bindable, oldValue, newValue)=>
+                {
+                    var colView = (CollectionView)bindable;
+                    var oldSelection = colView.selectedItems ?? s_empty;
+                    //FIXME : CoerceSelectedItems calls only isCreatedByXaml
+                    var newSelection = (SelectionList)CoerceSelectedItems(colView, newValue);
+                    colView.selectedItems = newSelection;
+                    colView.SelectedItemsPropertyChanged(oldSelection, newSelection);
+                },
+                defaultValueCreator: (bindable) =>
+                {
+                    var colView = (CollectionView)bindable;
+                    colView.selectedItems = colView.selectedItems ?? new SelectionList(colView);
+                    return colView.selectedItems;
+                });
+
+        /// <summary>
+        /// Binding Property of selected items list in multiple selection.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty SelectionModeProperty =
+            BindableProperty.Create(nameof(SelectionMode), typeof(ItemSelectionMode), typeof(CollectionView), ItemSelectionMode.None,
+                propertyChanged: (bindable, oldValue, newValue)=>
+                {
+                    var colView = (CollectionView)bindable;
+                    oldValue = colView.selectionMode;
+                    colView.selectionMode = (ItemSelectionMode)newValue;
+                    SelectionModePropertyChanged(colView, oldValue, newValue);
+                },
+                defaultValueCreator: (bindable) =>
+                {
+                    var colView = (CollectionView)bindable;
+                    return colView.selectionMode;
+                });
+
+
+        private static readonly IList<object> s_empty = new List<object>(0);
```

Comment:
Since last year, NUI is not using a character `_` for field/property/method name.
Could you remove `_`?

Reply:
All _ is removed excepts Template/ which is already implemented in NUI and only moved.
Do i have to remove _ in Template either?

And is there any policy about License attachment?
I saw most of file has attatched apache licence but some of them are missing.

Reply:
Removing `_` from your new code is enough. Yon don't have to clean up the existing code.

And please add apache license like other NUIU files :)